packages/ui/src/common/components/selects/select.tsx

```diff
@@ -46,6 +46,7 @@ export const Select = <T extends any, V extends any = T>({
 
   useEffect(() => {
     const clickListener = (event: MouseEvent) => {
+      // TODO: figure out a way to enable clicking on the links in Tooltips
```

Comment:
i hope it coul help (check for example - StakingAccountStep.tsx file). There exist tooltip with link